csep/utils/plots.py

```diff
@@ -1342,7 +1342,7 @@ def plot_comparison_test(results_t, results_w=None, axes=None, plot_args=None):
     xTickPos = ax.get_xticks()
     ax.yaxis.set_major_locator(matplotlib.ticker.MaxNLocator(integer=True))
     ax.set_ylim([ylim[0], ylim[1]])
-    ax.set_xlim([ax.get_xlim()[0] - 0.5, ax.get_xlim()[1] + 0.5])
+    ax.set_xlim([0.5, len(results_t) - 0.5])
```

Comment:
nope, this still hides me a forecast. It works for me with [ - 0.5, len(results_t) - 0.5].  Weiirdd.

Reply:
missed the '-'. catch you tomorrow! gn!